Code/Java/Tower_of_Hanoi.java

```diff
@@ -0,0 +1,83 @@
+
+/**
```

Comment:
add comments like 
/*

Do not add * in between the comments

Reply:
Changes made